tools/apiview/parsers/csharp-api-parser/CSharpAPIParserTests/ProgramTests.cs

```diff
@@ -0,0 +1,20 @@
+using CSharpAPIParser;
+
+namespace CSharpAPIParserTests
+{
+    public class ProgramTests
+    {
+        [Theory]
+        [InlineData("[2.3.24, 3.0.0)")]
```

Comment:
let's include a test case for every version range permutation - https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#version-ranges